push_notification/apps/notification/api_services/notification.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from ..services.notification_service import NotificationService
+from ..serializers.notification import NotificationSerializer
+from ..exceptions import NotificationNotFound
+
+logger = logging.getLogger(__name__)
+
+
+class NotificationAPIService(object):
```

Comment:
why are we building this api service layer ?

Reply:
I will be further create webpush to send notification to all subscribed subscriber, APIService will help to get required details